<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="whatsThis">
   <string comment="Search history" extracomment="Search history"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="topSearchLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Search word</string>
    </property>
   </widget>
   <widget class="QPushButton" name="topSearchPushButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
    <property name="icon">
     <iconset theme="edit-find">
      <normaloff>.</normaloff>.</iconset>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="historyListView">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>51</y>
      <width>181</width>
      <height>496</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="historySearchLineEdit">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string comment="Search history"/>
    </property>
    <property name="placeholderText">
     <string>Search history</string>
    </property>
   </widget>
   <widget class="QPushButton" name="donatePushButton">
    <property name="geometry">
     <rect>
      <x>705</x>
      <y>560</y>
      <width>86</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Donate</string>
    </property>
    <property name="icon">
     <iconset theme="emblem-favorite-symbolic">
      <normaloff>../../../</normaloff>../../../</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="aboutPushButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>560</y>
      <width>86</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>About</string>
    </property>
    <property name="icon">
     <iconset theme="help-about-symbolic">
      <normaloff>../../../</normaloff>../../../</iconset>
    </property>
   </widget>
   <widget class="QTabWidget" name="mainTabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>601</width>
      <height>546</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="dictLookUpTab">
     <attribute name="icon">
      <iconset theme="edit-find">
       <normaloff>.</normaloff>.</iconset>
     </attribute>
     <attribute name="title">
      <string>Dictionary Look Up</string>
     </attribute>
     <widget class="QTextEdit" name="dictLookUpTextEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>576</width>
        <height>496</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="wildcSearchTab">
     <attribute name="icon">
      <iconset theme="dialog-question-symbolic">
       <normaloff>.</normaloff>.</iconset>
     </attribute>
     <attribute name="title">
      <string>Wildcard Word Search</string>
     </attribute>
     <widget class="QListView" name="wildcMatchesListView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>181</width>
        <height>466</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="wildcSearchLookUpTextEdit">
      <property name="geometry">
       <rect>
        <x>205</x>
        <y>40</y>
        <width>381</width>
        <height>466</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="wildcMatchesLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Wildcard matches:</string>
      </property>
     </widget>
     <widget class="QLabel" name="wildcLookUpLabel">
      <property name="geometry">
       <rect>
        <x>205</x>
        <y>10</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Look up:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="wikitionaryTab">
     <attribute name="icon">
      <iconset theme="text-html-symbolic">
       <normaloff>.</normaloff>.</iconset>
     </attribute>
     <attribute name="title">
      <string>Wikitionary</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="historyPurgePushButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="edit-delete-symbolic">
      <normaloff>.</normaloff>.</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
